src/olympia/editors/templates/editors/includes/files_view.html

```diff
@@ -1,5 +1,6 @@
 <div><strong>{{ _('Files in this version:') }}</strong></div>
 <ul>
+{% set latest_version=addon.find_latest_version(channel=amo.RELEASE_CHANNEL_LISTED) %}
```

Comment:
so instead, this ?
`{% set latest_version_in_channel=addon.find_latest_version(channel=version.channel) %}` ?

(Assuming this template does receive a `version`, but I can add it if it doesn't)